pkg/ocp4/ocp4.go

```diff
@@ -50,18 +60,47 @@ func (ocp4Node *Node) Translate(ocp3Node configv1.NodeConfig) {
 // GenYAML returns the list of translated CRDs
 func (ocp4Master *Master) GenYAML() ([]Manifest, error) {
 	var manifests []Manifest
-	if ocp4Master.OAuth.Kind != "" {
-		manifest := Manifest{Name: "100_CPMA-cluster-config-oauth.yaml", CRD: ocp4Master.OAuth.GenYAML()}
+
+	// Generate yaml for oauth config
+	crd := ocp4Master.OAuth.GenYAML()
+	manifests = appendOauthManifests(ocp4Master.OAuth.Kind, crd, manifests)
```

Comment:
Name are like color and taste meanwhile a good practice is to avoid generic content.
Maybe just `OAuthManifests`?